src/Sarif.Converters/Sarif.Converters.csproj

```diff
@@ -37,6 +37,19 @@
     <EmbeddedResource Include="Schemata\FxCopReport.xsd" />
   </ItemGroup>
 
+  <Choose>
+    <When Condition="$(TargetFramework) == 'netstandard2.0'">
+      <ItemGroup>
+        <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+      </ItemGroup>
+    </When>
+    <Otherwise>
+      <ItemGroup>
+        <PackageReference Include="Newtonsoft.Json" Version="8.0.1" />
```

Comment:
>8.0.1 [](http://example.com/codeflow?start=61&length=5)

here we are not using 6, but 8 
I see the reason is we have a attribute that does not exists in 8 and will not build.
If not for this reason we will be also using 6 like other projects.
Let me know if I understand correctly.

Reply:
Yes, exactly. The HDF JSON definitions in particular consume a new attribute enumeration property that doesn't exist in down-level Newtonsoft.Json (it is an attribute that specifies that a property is *not* required but also cannot be null when present). Accepting 6.0.8 support would have required an intrusive change to resolve this and also would have compromised enforcement for this specific serialization condition. We can discuss more offline, now that this PR is merged.

Reply:
Accepting 6.0.8 support would have required an intrusive change to resolve  --- agree, just wanted to make sure I understand the reason of the choice we made.